plugins/obs-transitions/data/pixelate_transition.effect

```diff
@@ -0,0 +1,96 @@
+// Written by Xaymar.
+// https://www.shadertoy.com/view/MtdXDM
+// Translated/adapted to OBS by c3r1c3.
+
+uniform float4x4 ViewProj;
+uniform texture2d tex_a;
+uniform texture2d tex_b;
+// Transition duration/time allotment.
+uniform float pixelate_val;
+
+// Configurable
+//float transitionTime = 1.5; // Transition lasts x Seconds.
+//float maximumBlockSize = 9.0; // Maximum Block Size (2 ^ x)
+//float2 blockOffset = float2(0.5, 0.5); // Inner Block offset
+//float2 pixelateCenter = float2(0.5, 0.5); // Pixelate offset.
+
+sampler_state textureSampler {
+	Filter   = Linear;
+	AddressU = Clamp;
+	AddressV = Clamp;
+};
+
+struct VertData
+{
```

Comment:
Sorry, I have no idea... (well i do know how) this file got in there. Removing it.